description: fix formatting calls that might overflow (gcc -Wformat-overflow)
author: Michael Gilbert <mgilbert@debian.org>

--- a/tools/winedump/msc.c
+++ b/tools/winedump/msc.c
@@ -461,7 +461,7 @@ static const char* get_callconv(unsigned
     case CV_CALL_RESERVED:      callconv = "reserved"; break;
     default:
         {
-            static char tmp[16];
+            static char tmp[18];
             sprintf(tmp, "callconv=%x", cc);
             callconv = tmp;
         }
